int64 motor1
int64 motor2
---
string Sum
